project-management/src/main/java/life/qbic/projectmanagement/application/measurement/MeasurementService.java

```diff
@@ -317,54 +238,31 @@ protected List<MeasurementId> performRegistration(
   private List<MeasurementId> performRegistrationNGS(
       List<MeasurementMetadata> measurementMetadataList,
       ProjectId projectId) {
-    List<NGSMeasurementMetadata> ngsMeasurements = new ArrayList<>();
+    List<NGSMeasurementMetadata> ngsMeasurementMetadata = new ArrayList<>();
     for (MeasurementMetadata measurementMetadata : measurementMetadataList) {
       if (measurementMetadata instanceof NGSMeasurementMetadata) {
-        ngsMeasurements.add((NGSMeasurementMetadata) measurementMetadata);
+        ngsMeasurementMetadata.add((NGSMeasurementMetadata) measurementMetadata);
       }
     }
-    Map<NGSMeasurement, Collection<SampleIdCodeEntry>> ngsMeasurementsMapping = new HashMap<>();
-    for (NGSMeasurementMetadata metadata : ngsMeasurements) {
-      ngsMeasurementsMapping.putAll(prepareNGSMeasurement(projectId, metadata));
-    }
-    return measurementDomainService.addNGSAll(ngsMeasurementsMapping);
-  }
+    Map<NGSMeasurement, Collection<SampleIdCodeEntry>> NGSmeasurementsMapping = new HashMap<>();
```

Comment:
variable names should start with a lower case letter. 
```suggestion
    Map<NGSMeasurement, Collection<SampleIdCodeEntry>> ngsMeasurementsMapping = new HashMap<>();
```